atspi-common/src/object_reference.rs

```diff
@@ -8,17 +8,17 @@ pub const ACCESSIBLE_PAIR_SIGNATURE: Signature<'_> = Signature::from_static_str_
 // check where the lifetimes of the borrow are tied to, see also: comment on `interface()` method
 // in `DefaultEvent` impl
 // then rename into Owned for this one.
-/// Owned Accessible type
+/// Owned `ObjectReference` type
 /// Emitted by `CacheRemove` and `Available`
 #[derive(Debug, Clone, Serialize, Deserialize, Type, PartialEq, Eq, Hash)]
-pub struct Accessible {
+pub struct ObjectReference {
```

Comment:
Unless it was previously ruled out, I'd suggest renaming this struct `ObjectRef`. The name is quite long for a type so pervasive, and the `Ref` keyword should be known by most Rust developers.